src/core/sys/windows/sdkddkver.d

```diff
@@ -0,0 +1,155 @@
+enum _WIN32_IE_WINBLUE                =  _WIN32_IE_IE100;
+
+
+//
+// NTDDI version enumants
+//
+enum NTDDI_WIN2K                         = 0x05000000;
+enum NTDDI_WIN2KSP1                      = 0x05000100;
+enum NTDDI_WIN2KSP2                      = 0x05000200;
+enum NTDDI_WIN2KSP3                      = 0x05000300;
+enum NTDDI_WIN2KSP4                      = 0x05000400;
+
+enum NTDDI_WINXP                         = 0x05010000;
+enum NTDDI_WINXPSP1                      = 0x05010100;
+enum NTDDI_WINXPSP2                      = 0x05010200;
+enum NTDDI_WINXPSP3                      = 0x05010300;
+enum NTDDI_WINXPSP4                      = 0x05010400;
+
+enum NTDDI_WS03                          = 0x05020000;
+enum NTDDI_WS03SP1                       = 0x05020100;
+enum NTDDI_WS03SP2                       = 0x05020200;
+enum NTDDI_WS03SP3                       = 0x05020300;
+enum NTDDI_WS03SP4                       = 0x05020400;
+
+enum NTDDI_WIN6                          = 0x06000000;
+enum NTDDI_WIN6SP1                       = 0x06000100;
+enum NTDDI_WIN6SP2                       = 0x06000200;
+enum NTDDI_WIN6SP3                       = 0x06000300;
+enum NTDDI_WIN6SP4                       = 0x06000400;
+
+enum NTDDI_VISTA                       = NTDDI_WIN6;
+enum NTDDI_VISTASP1                    = NTDDI_WIN6SP1;
+enum NTDDI_VISTASP2                    = NTDDI_WIN6SP2;
+enum NTDDI_VISTASP3                    = NTDDI_WIN6SP3;
+enum NTDDI_VISTASP4                    = NTDDI_WIN6SP4;
+
+enum NTDDI_LONGHORN= NTDDI_VISTA;
+
+enum NTDDI_WS08                        = NTDDI_WIN6SP1;
+enum NTDDI_WS08SP2                     = NTDDI_WIN6SP2;
+enum NTDDI_WS08SP3                     = NTDDI_WIN6SP3;
+enum NTDDI_WS08SP4                     = NTDDI_WIN6SP4;
+
+enum NTDDI_WIN7                          = 0x06010000;
+enum NTDDI_WIN8                          = 0x06020000;
+enum NTDDI_WINBLUE                       = 0x06030000;
+
+
+//
+// masks for version macros
+//
+enum OSVERSION_MASK      = 0xFFFF0000;
+enum SPVERSION_MASK      = 0x0000FF00;
+enum SUBVERSION_MASK     = 0x000000FF;
+
+enum  _WIN32_WINNT   = 0x0603;
+
+enum NTDDI_VERSION   = 0x06030000;
+enum WINVER       =  _WIN32_WINNT;
+
+// set _WIN32_IE based on _WIN32_WINNT
+static if (_WIN32_WINNT <= _WIN32_WINNT_NT4)
+   enum _WIN32_IE    =  _WIN32_IE_IE50;
+else static if (_WIN32_WINNT <= _WIN32_WINNT_WIN2K)
+   enum _WIN32_IE    =  _WIN32_IE_IE501;
+else static if (_WIN32_WINNT <= _WIN32_WINNT_WINXP)
+   enum _WIN32_IE    =  _WIN32_IE_IE60;
+else static if (_WIN32_WINNT <= _WIN32_WINNT_WS03)
+   enum _WIN32_IE    =  _WIN32_IE_WS03;
+else static if (_WIN32_WINNT <= _WIN32_WINNT_VISTA)
+   enum _WIN32_IE    =  _WIN32_IE_LONGHORN;
+else static if (_WIN32_WINNT <= _WIN32_WINNT_WIN7)
+   enum _WIN32_IE    =  _WIN32_IE_WIN7;
+else static if (_WIN32_WINNT <= _WIN32_WINNT_WIN8)
+   enum _WIN32_IE    =  _WIN32_IE_WIN8;
+else
+   enum _WIN32_IE       = 0x0A00;
```

Comment:
I think we already have this here:

https://github.com/dlang/druntime/blob/d7800b516fd31eb980fb64ce99d02f22c75820d9/src/core/sys/windows/w32api.d#L48-L84